doc/DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/zmtl-SPECTROGRAPH-TILEID-GROUPID.rst

```diff
@@ -69,35 +71,35 @@ Required Data Table Columns
 ==================== ======= ===== ===================
 Name                 Type    Units Description
 ==================== ======= ===== ===================
-RA                   float64 deg   label for field   1
-DEC                  float64 deg   label for field   2
-TARGETID             int64         label for field   3
-SV1_DESI_TARGET [1]_ int64         label for field   4
```

Comment:
You can't actually delete these columns.  Because if they are *absent* from the data model then the data model verification code will flag their presence in a data file as an error.  However if they are marked *optional* then their presence or absence in a data file will not raise any particular issue.

Reply:
Noted, thanks. I'll put those back in.

Reply:
I can change the message to something more informative than just "Optional" though, right? As in _why_ these columns are optional? i.e. the data model verification code is just looking for the existence of the columns, not for the word "Optional," correct?

Reply:
Oops, I accidentally started a review there. Please answer the question, though, I'll cancel that review.

Reply:
I can change the message to something more informative than just "Optional" though, right? As in why these columns are optional? i.e. the data model verification code is just looking for the existence of the columns, not for the word "Optional," correct?

Reply:
You cannot change the notation on the column itself: `[1]_`, because that is needed by the code.  However if you want to change `.. [1] Optional` to something else, that's fine.

Reply:
OK, I restored these columns.

Reply:
Looks good, thank you!